ontology/uco/vocabulary/vocabulary.ttl

```diff
@@ -11,9 +11,15 @@
 
 vocabulary:AccountTypeVocab
 	a rdfs:Datatype ;
-	rdfs:subClassOf rdfs:Resource ;
 	rdfs:label "Account Type Vocabulary"@en-US ;
-	owl:oneOf vocabulary:AccountTypeVocab-members ;
+	owl:equivalentClass [
+		a rdfs:Datatype ;
+		owl:onDatatype xsd:string ;
+		owl:withRestrictions (
+			owl:oneOf
+			vocabulary:AccountTypeVocab-members
```

Comment:
Ah, @frederich-stine , I apologize I apparently misspoke when I described this to you.

The syntax change should start from the Feature-Issue-406 branch, not start from `develop` and merge into the 406 branch.  There is a correction in the 406 branch that you will not want to spend time repeating.

Reply:
First, you're caught up with the test branch.  Thanks for the merge!

But, there's another syntax error here.  The OWL-to-RDF mapping doesn't quite make some constraints' separations clear, but there is a demonstration in the OWL Primer.  The short of it is, `:withRestrictions` should not be used to wrap `owl:oneOf`.  `owl:Restriction`s are for constraining literals in non-enumerative manners, like requiring integers be greater than a certain value, or strings match a regular expression.

You can see this demonstrated in the Primer with ["Show Turtle Syntax" turned on](https://www.w3.org/TR/owl2-primer/#OWL_Syntaxes).  Then search for `:withRestrictions` and `owl:oneOf` (esp. `:todderAge`).

If you just cut the `owl:withRestrictions (` and closing `)`, `make check` will create a re-normalized file for you and save you a lot of `\t` deletions.

Reply:
Makes sense!
Have removed the `owl:withRestrictions` from the enumerated values. Tests passing again.

Reply:
This thread's settled.  Thanks again, @frederich-stine !